src/plugins/htmlVideoPlayer/plugin.js

```diff
@@ -1366,55 +1572,64 @@ define(['browser', 'require', 'events', 'apphost', 'loading', 'dom', 'playbackMa
                         // don't animate on smart tv's, too slow
                         if (options.fullscreen && browser.supportsCssAnimation() && !browser.slow) {
                             zoomIn(dlg).then(function () {
```

Comment:
This will fix forced transcoding.
```suggestion
                            return zoomIn(dlg).then(function () {
```

Reply:
Okay cool, will try that, I probably wouldn't have found this